### DISCUSSION TITLE
How to calculate noise propagation from aircraft noise source ?

### DISCUSSION THREAD

**Discussion by solarblindfull** [Universite-Gustave-Eiffel/NoiseModelling#324] [Q&A]
Hello and thank you very much for this software.

I want to make a noise map about the propagation of the noises emissions from aircrafts in urban areas around the Paris - Charles-de-Gaulle french airport. I have flights RADAR datas but I can't find an open data software that can simulate this kind of noises sources.

Until now, I can't find a way to integrate custom noises sources in the last version of your software (v.3.3.2). Have you a trick to do it or have you plan to integrate that feature soon in the next release ?

Cordially

**Comment by pierromond** [comment]
Hello and thank you for your message! 

According to CNOSSOS, taking into account aircrafts is a very special development, and it is not implemented in the software. 

Nevertheless, as a first approximation you can always "make source points flying". You can find the following block in the experimental folder: "https://github.com/Ifsttar/NoiseModelling/blob/cnossos/wps_scripts/src/main/groovy/org/noise_planet/noisemodelling/wps/Experimental/Drone_Dynamic_map.groovy". From this block you can retrieve the sound level at the receiver points of the flying source points. We tried it to estimate the impact of drone flights. You will probably have to adapt it to your case, but it is possible. Keep us informed !

About point sources : #207 #137 #191

**Reply by solarblindfull** [reply]
Thank you for your answer !

I understand, it's much complicated that I expected. My first attempt was to create aircraft trajectory like linears but I didn't find a way to implement altitudes and aircraft noise emission maths into that lines attributes. 
What is this "block" system you describe ? How it links to the shapes attributes ? What parameters I need to modify for altitudes and maths in the drone block ? Sorry for the begginer questions, I'm a student.

I will try too adapt your drone solution and see what will happen.

Cordially
